test/sim/abilities/liquidooze.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Liquid Ooze', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it('should damage the target by the amount it would have healed after it uses a draining move', function () {
+		battle = common.createBattle([[
+			{species: 'tentacruel', ability: 'liquidooze', moves: ['sleeptalk']},
+		], [
+			{species: 'serperior', moves: ['gigadrain']},
+		]]);
+		battle.makeChoices();
+		const damage = battle.p2.active[0].maxhp - battle.p2.active[0].hp;
+		assert.bounded(damage, [27, 33]);
```

Comment:
For each of these tests, I think it'd be better just to do ``assert.fullHP()`` or ``assert.false.fullHP()``; we don't really care about the particular damage the target is at for any of these, just that it was damaged.